workshop/exercise-3/README.md

```diff
@@ -4,7 +4,9 @@ Let's get our first Knative application up & running. Using the Build & Serving
 
 
 ### Update domain configuration for Knative
-When a Knative application is deployed, Knative will define a URL for your application. By default, this url is "example.com." Because we want our application to be accessible at a URL we own, we need to configure Knative to assign new applications to our own hostname.
+When a Knative application is deployed, Knative will define a URL for your application. By default, Knative Serving routes use `example.com` as the default domain. The fully qualified domain name for a route by default is `{route}.{namespace}.{default-domain}`.
```

Comment:
I wonder if we need to make it cleat that "route" in this case means the "app name"?